### DISCUSSION TITLE
Unable to do a checkout in the Unreal menu

### DISCUSSION THREAD

**Discussion by FermiPerumal** [ProjectBorealis/UEGitPlugin#145] [Q&A] [answered]
Hi, I have an Unreal project set up with Git LFS 2 module connected to gitlab. The project has Git LFS enabled and I also enabled the verify locks config option. The commit and check in processes work but the Checkout option is disabled. How do I get this working?

Thanks!

![image](https://github.com/ProjectBorealis/UEGitPlugin/assets/17150545/d05d1bd0-0ce2-4adf-aaa3-bd74c81ad23f)

**Comment by mastercoms** [comment]
whats your gitattributes

**Reply by FermiPerumal** [reply]
@mastercoms 
This is the contents of my .gitattributes file:
![image](https://github.com/ProjectBorealis/UEGitPlugin/assets/17150545/52ed6ae2-8dfd-493a-9c28-fa83eca06146)

**Reply by mastercoms** [reply, marked as answer]
https://github.com/ProjectBorealis/UEGitPlugin?tab=readme-ov-file#note-about-gitattributes-and-gitignore

**Reply by FermiPerumal** [reply]
That worked! Thank you so much.

**Reply by FermiPerumal** [reply]
@mastercoms For some reason after I check out, I am unable to release the lock. Check in or Revert button does not help. Once it's LFS-locked, I am not able to release it. Can you pls help?

**Reply by mastercoms** [reply]
Make sure in your revision control settings you have the correct LFS username set.

**Reply by FermiPerumal** [reply]
@mastercoms Seems like after a restart, the Revert option works!

**Reply by mastercoms** [reply]
Awesome!

**Reply by FermiPerumal** [reply]
@mastercoms For some reason, now when I check out, make changes and save the uasset, we are unable to see the changed asset in Submit Content UI. It's showing up in an external Ui like source tree as modified, but not within Unreal. How do I fix this?

**Reply by mastercoms** [reply]
So it was working before but now without anything changing it doesn't work anymore? Does it always happen for any type of file or just a certain operation/file type?

**Reply by FermiPerumal** [reply]
It used to work but not anymore. It is not working for any files: umaps, uassets etc.. Check out happens and the lock is present. Changes are also getting saved without any issues. Just that the UI is not picking it up

**Reply by mastercoms** [reply]
So you haven't updated the plugin or anything? Also does check in present as an option in the right click menu?

**Reply by FermiPerumal** [reply]
Yes check in is an option in the right-click menu as well. Nothing changed in the plugin, it just stopped recognizing the modified files

**Reply by mastercoms** [reply]
I'm really not sure what's going on to cause that, sorry. Does it happen for other people on your project? Does it happen with a fresh clone of the project? Besides these basic troubleshooting steps, I have tried to figure out what could be the issue but haven't been able to find anything thus far. I'll keep you posted if I find anything on my end.

**Reply by FermiPerumal** [reply]
No problem, I'm confused as to how it stopped working too. I'll keep digging. Thank you anyways for your support!
